packages/postgres-database/src/simcore_postgres_database/models/comp_run_collections.py

```diff
@@ -0,0 +1,40 @@
+"""Computational Runs Table"""
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+
+comp_runs_collections = sa.Table(
+    "comp_runs_collections",
+    metadata,
+    sa.Column(
+        "collection_run_id",
+        UUID(as_uuid=True),
+        server_default=sa.text("gen_random_uuid()"),
+        nullable=False,
+        primary_key=True,
+    ),
+    sa.Column(
+        "client_or_system_generated_id",
```

Comment:
What is client? what is system?
whare are they generating?
I think it is worth to document a bit of context here. Think of an agent trying to make sense of this schema. What would you document?